qutebrowser/config/configdata.py

```diff
@@ -1638,8 +1638,8 @@ def data(readonly=False):
         ('move-to-end-of-line', ['$']),
         ('move-to-start-of-document', ['gg']),
         ('move-to-end-of-document', ['G']),
-        ('yank-selected -p', ['Y']),
-        ('yank-selected', ['y'] + RETURN_KEYS),
+        ('yank selection -p', ['Y']),
```

Comment:
Looking at 489c913e586a37a6e982ddf152d6ed7fb6984b8c that should probably be `-s` - I'm guessing @NoctuaNivalis mixed up `-s` (**s**election) and `-p` (**p**rimary) and I didn't catch it.